### PR TITLE
Add bank account model

### PR DESCRIPTION
**Pull request by neungkl**

### CODE REVIEW

**Comment by neungkl** [comment]
Please merge #81 before this PR

File: webserver/apps/bank_accounts/models.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from apps.lessors.models import Lessor
+
+# Create your models here.
+class BankAccount(models.Model):
+    BANK_CHOICES = (
+        ('BBL', 'Bankok Bank'),
```
Comment (futurizing) [inline webserver/apps/bank_accounts/models.py]: Bangkok Bank

File: webserver/apps/bank_accounts/models.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from apps.lessors.models import Lessor
+
+# Create your models here.
```
Comment (futurizing) [inline webserver/apps/bank_accounts/models.py]: Remove

File: webserver/apps/bank_accounts/models.py (hunk 1 above)
Comment (neungkl) [reply webserver/apps/bank_accounts/models.py]: Fixed

File: webserver/apps/bank_accounts/models.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        ('SCB', 'The Siam Commercial Bank'),
+        ('NBANK', 'Thanachart Bank'),
+        ('SCIB', 'Siam City Bank'),
+        ('GSB', 'Government Savings Bank'),
+        ('GHB', 'Government Housing Bank'),
+    )
+    account_name = models.CharField(verbose_name='Account Name', max_length=30)
+    account_id = models.CharField(verbose_name='Account ID', max_length=20, unique=True)
+    bank = models.CharField(verbose_name='Bank', max_length=5, choices=BANK_CHOICES)
+    branch = models.CharField(verbose_name='Branch', max_length=20)
```
Comment (futurizing) [inline webserver/apps/bank_accounts/models.py]: Is 20 enough?

File: webserver/apps/bank_accounts/models.py (hunk 2 above)
Comment (neungkl) [reply webserver/apps/bank_accounts/models.py]: Fixed

File: webserver/apps/bank_accounts/models.py (hunk 3 above)
Comment (neungkl) [reply webserver/apps/bank_accounts/models.py]: Changed to 100

**Pull request merged**